build.gradle

```diff
@@ -200,7 +216,7 @@ configurations {
 }
 
 platform {
-	fetchSources = true
+	fetchSources = false
```

Comment:
Can it be made to work with sources easily?
If not it would be good to comment here what the blocker is.

Reply:
It works with sources enabled, I have updated the changes.